### PR TITLE
Add docs of e2e development guide

### PR DESCRIPTION
**Pull request by kezhenxu94**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: docs/en/guides/README.md [hunk 1]
```diff
@@ -39,10 +40,52 @@ Therefore, to run the UTs, try `./mvnw clean test`, this will only run the UTs,
...
+ 
+In order to make it more easily to orchestrate, we've provided a [Docker e2e-container](https://github.com/SkyAPMTest/e2e-container) that can run
+OAP server in both standalone and cluster mode, it can also run the given instrumented services. Refer to the repository for more details;
+
+Basically you will need:
+1. (if in cluster mode) start up the third-party service containers that SkyWalking depends on, such as ZooKeeper as coordinator, ElasticSearch as storage, etc. in the docker-maven-plugin;
+1. (if in cluster mode) carefully map the addresses/ports into the e2e-container that can be used by the OAP server to connect to;
+1. start the OAP server, Web App, and instrumented services in the e2e-container, mount your customized startup script to `/rc.d/` in e2e-container and it gets run when the container starts up;
+1. use the utilities provided in e2e-container to check the healthiness of the components;
+1. design the test controller and verify expected result;
```
Comment (wu-sheng) [inline docs/en/guides/README.md]: Make the first letter uppercase.

File: docs/en/guides/README.md [hunk 2]
```diff
@@ -39,10 +40,52 @@ Therefore, to run the UTs, try `./mvnw clean test`, this will only run the UTs,
...
+The e2e test is an individual project under the SkyWalking root directory and the IDEA cannot recognize it by default, right click
+on the file `test/e2e/pom.xml` and click `Add as Maven Project`, things should be ready now.
+
+- Orchestrate the components
+
+Our goal of E2E tests is to test the SkyWalking project in a whole, including the OAP server, Web App, and even the frontend UI(not now),
+in single node mode as well as cluster mode, therefore the first step is to determine what case we are going to verify and orchestrate the 
+components.
+ 
+In order to make it more easily to orchestrate, we've provided a [Docker e2e-container](https://github.com/SkyAPMTest/e2e-container) that can run
```
Comment (wu-sheng) [inline docs/en/guides/README.md]: Not `have provided`, it is `using` since we haven't donated it to Apache.

**Pull request merged**